### PR TITLE
Use eth_labels_final table in labels query

### PR DESCRIPTION
**Pull request by ninjascant**
## Changes

Use `eth_labels_final` table instead of `blockchain_address_labels` to obtain address labels. This will allow to speed up query

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by tspenov** [comment]
For some reason I get `DB::Exception: Table default.eth_labels_final doesn't exist` when testing locally. I see there is such table when I connect to clickhouse. The only difference from querying from sanbase is that we use `clickhouse-proxy` and other user. We can deploy and test on stage with this query:
```
{
  projectBySlug(slug:"santiment") {
    tokenTopTransactions(from: "utc_now-1w", to: "utc_now", limit:100) {
      fromAddress {
        address
        labels {
          name
        }
      }
      toAddress {
        address
        labels {
          name
        }
      }
      trxValue
      trxHash
    }
  }
}
```
to see whether we have proper labels.

**Comment by tspenov** [comment]
After adding `eth_labels_final` to user facing cluster - it works.

**Review by tspenov — APPROVED** [review]

File: lib/sanbase/clickhouse/metric/metric_files/eth2_metrics.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "human_readable_name": "ETH2.0 Staker Count",
+    "name": "eth2_staker_count",
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/metric/metric_files/eth2_metrics.json:4]: Let's expose it as `eth2_stakers_count` (plural) to be consistent with other `*_count` metrics

Comment (Snedashkovsky) [reply lib/sanbase/clickhouse/metric/metric_files/eth2_metrics.json]: Updated

**Pull request merged**